docs/v1/P4Runtime-Spec.mdk

```diff
@@ -3996,9 +4028,18 @@ A `WriteRequest` may only include an `Update` message of type `MODIFY` with a
   specified by the client. Note that if this Protobuf field is not set, the
   meter config is set to execute the default behavior (GREEN for all packets).
 
+* `counter_data` is used to set the per color counter values to the default
+  value of (0), which is essentially clearing the counters. If the field is
+  unset, the counter values are left untouched. If the field is set, targets
+  supporting this counter should reset all the color counters to (0),
```

Comment:
> targets supporting this counter should reset all the color counters to (0).

It depends. If any of the subfields are nonzero, targets supporting the counter should presumably return INVALID_ARGUMENT, to be consistent with the spec for TableEntry?